script/nan-spec-runner.js

```diff
@@ -128,7 +124,7 @@ async function main () {
     return process.exit(installStatus !== 0 ? installStatus : signal);
   }
 
-  const onlyTests = args.only && args.only.split(',');
+  const onlyTests = args?.only.split(',');
```

Comment:
```suggestion
  const onlyTests = args.only?.split(',');
```

Think the question mark migrated a little too far left, CI is failing on this line.